packages/ava/src/text/utils/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './isType';
+export * from './arr2map';
```

Comment:
shall we put these utils just under src/ ?

Reply:
I think it is necessary, I will put it over.